### ISSUE TITLE
(BUG) Automations do not appear to work

### ISSUE THREAD

**Issue by drizuid** [tonesto7/echo-speaks#350]
## Verify the following before opening an trouble issue

**Go over all the following points, and put an `x` in all the boxes that apply.
If you're unsure about any of these, don't hesitate to ask. We're here to help!**

-  [X] That _OAuth_ is Enabled for the SmartApp under the IDE.
-  [X] The App/SmartApp and Device Handler/Driver are using the latest code available.
-  [X] You have reviewed the [Echo Speaks Documentation](https://tonesto7.github.io/echo-speaks-docs) for potential fixes.
-  [X] That Both the Apps/SmartApps and Device Handlers/Drivers have been _Published for You_ in the IDE.

---
## About Your Setup
-  Hub Platform (SmartThings or Hubitat): Smartthings
-  How many devices are detected?: 9
-  iOS or Android?: android
-  Mobile App Version(Not required): 
-  App/SmartApp Version: 3.6.3.1
-  Device Handler/Driver Version: 3.6.3.1
-  Heroku Server Version: 2.5.2

## Expected Behavior
When the trigger requirements are met, the action should occur

## Current Behavior
Nothing happens

## Steps to Reproduce (for bugs)
**Provide a link to a live example, or an unambiguous set of steps to reproduce this bug. Include code to reproduce, if relevant**

1. Create action (see below)
2. test this action OR perform action
3.nothing happens

## Context
**action type:** speak
**action trigger:** contacts/doors/windows - all changes to any, custom contact response `<speak><prosody pitch="x-high" volume="x-loud">wop, wop, wop</prosody>%name %type is now %value%</speak>`
**execution config:** echo speaks devices (4 echo devices)

when i click test this action then hit next, it fails with "We're having trouble connecting. Check your network connection and try again."

If I do an actual test (open the door/window) ST changes state but the action is not acted upon.

putting the same <speak> into device testing -> speech test works without an issue.
As an added note, the alexa guard control also does not work. While device testing works fine, nothing else appears to work.

basically, if a door/window is open, i want to hear beep beep beep <door/window name> is <open|closed> via my alexa devices

---

logging: enabled debug and detailed logs and emailed direct. Doesn't show much though.

**Comment by drizuid** [comment]
a little video to demonstrate some of the odd behavior
https://youtu.be/7lfYfKRRsOg

**Comment by drizuid** [comment]
upgraded to the latest and after a couple hours it started working. the alexa guard thing still isn't working, but the title of this issue is actions, so closing. thanks!
